app/scripts/directives/notifications/notificationDrawerWrapper.js

```diff
@@ -99,6 +99,7 @@
         _.each(drawer.notificationGroups, function(group) {
           _.remove(group.notifications, { uid: notification.uid, namespace: notification.namespace });
         });
+        delete notificationsMap[$routeParams.project][notification.uid];
```

Comment:
@benjaminapetersen Is this a bug in remove? It seems like the `_.each` above should take care of this

Reply:
The notificationsMap is different from the group.notifications.  notificationsMap actually generates the group.notifications when switching projects, so it needs to be cleared in the map as well

Reply:
I understand, thanks.

Reply:
I would think so as well, looking.